examples/grpc/common/src/main/java/io/helidon/grpc/examples/common/JavaMarshaller.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * IMPORTANT NOTE: JavaMarshaller is useful for quick prototypes, but it is not recommended for production use. For
+ * one, it only supports gRPC-based communication between Java clients and servers, which is quite limiting. It is also
+ * not very efficient, and last but not least, it introduces security vulnerabilities.
+ */
+
+package io.helidon.grpc.examples.common;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import io.helidon.grpc.core.MarshallerSupplier;
+
+import io.grpc.MethodDescriptor;
+import jakarta.inject.Named;
+
+/**
+ * An implementation of a gRPC {@link MethodDescriptor.Marshaller} that
+ * uses Java serialization for testing.
+ *
+ * @param <T>  the type of value to be marshalled
+ */
+public class JavaMarshaller<T>
```

Comment:
Do we really need to use a Java marshaller? We actually wanted to remove any serialization from Helidon, and this is just re-introducing it into examples. 
We should use a different serialization approach in examples (proto, json) instead

Reply:
Removed and replaced with built-in JsonBMarshaller